src/components/Contacts.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+import { FaInstagram, FaFacebookF, FaTelegramPlane } from "react-icons/fa";
+import { fredoka } from "@/app/page";
+
+export default function Contacts() {
+  return (
+    <footer id="contacts" className="contacts">
+      <div className="contacts-section basis-full lg:basis-1/2 lg:self-center">
+        <p>
+          <span
+            className={`${fredoka.className} font-medium text-lg md:text-2xl text-primary`}
+          >
+            mariam crochet jewelry
+          </span>{" "}
+          is a Ukrainian clothing and crocheted jewelry brand founded by
+          designer Maryna Kambur. Handcrafted lovingly in Kyiv, Ukraine.
+        </p>
+      </div>
+      <div className="contacts-section">
+        <span className="contacts-subheader">Menu</span>
+        <Link href={"/about"}>About</Link>
+        <Link href={"/shop"}>Shop</Link>
+        <Link href={"/orders-info"}>Orders and delivery</Link>
+      </div>
+      <div className="contacts-section">
+        <span className="contacts-subheader">Contacts</span>
+        <div className="flex flex-row gap-4">
+          <a
+            href="https://www.instagram.com/mariam_crochet_jewelry/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <FaInstagram size={20} />
+          </a>
```

Comment:
I suggest to create a file
`src/constants/contacts.ts`
and add common contact constants there:
```ts
export const INSTAGRAM_LINK = '...'
export const PHONE_NUMBER = '...'        
```

```suggestion
          <a
            href={INSTAGRAM_LINK}
            target="_blank"
            rel="noopener noreferrer"
          >
            <FaInstagram size={20} />
          </a>
```